DesignSafe-CI/portal
# task/DES-1881: Social Media Icons

## Overview: ##
- Adds social media icons to the header
- Fixes NHERI link
- Fixes background color issue

## PR Status: ##


## Related Jira tickets: ##

* [DES-1881](https://jira.tacc.utexas.edu/browse/DES-1881)
* [DES-1840](https://jira.tacc.utexas.edu/browse/DES-1840)

## Testing Steps: ##
1. Start your containers and run `./mange.py collectstatic -i demo` in `des_django`
2. Load up designsafe.dev locally, and confirm your header matches the screenshots

## UI Photos:
- Collapsed:
<img width="743" alt="Screen Shot 2021-02-12 at 5 28 19 PM" src="https://user-images.githubusercontent.com/20326896/107833196-29c10f80-6d58-11eb-9630-727f78bac239.png">

- Fullscreen:
<img width="1791" alt="Screen Shot 2021-02-12 at 5 28 08 PM" src="https://user-images.githubusercontent.com/20326896/107833198-2a59a600-6d58-11eb-94b4-8fcb2439cc5e.png">
